front/src/config/i18n/fr.json

```diff
@@ -1760,6 +1828,41 @@
       "stop": "Stop",
       "record": "Enregistrer"
     },
+    "energy-sensor": {
+      "shortCategoryName": "Capteur d'énergie",
+      "voltage": "Tension",
+      "current": "Intensité",
+      "binary": "Commutateur",
+      "power": "Puissance",
+      "energy": "Energie",
+      "index": "Index"
+    },
+    "volume-sensor": {
+      "shortCategoryName": "Capteur de volume",
+      "decimal": "Volume (décimale)",
+      "integer": "Volume (entier)"
+    },
+    "currency": {
+      "shortCategoryName": "Monnaie",
+      "cost": "Coût"
+    },
+    "speed-sensor": {
+      "shortCategoryName": "Capteur de vitesse",
+      "decimal": "Vitesse"
+    },
+    "precipitation-sensor": {
+      "shortCategoryName": "Capteur de précipitations",
+      "decimal": "Précipitation"
+    },
+    "uv-sensor": {
+      "shortCategoryName": "Capteur d'UV",
+      "integer": "Indice UV"
+    },
+    "time-sensor": {
```

Comment:
"time-sensor" is weird, duration is better!

Reply:
I agree